frontend/src/components/EditResume.jsx

```
@@ -0,0 +1,672 @@
+  };
+
+  useEffect(() => {
+    calculateCompletion();
+  }, [resumeData]);
+
+   const validateAndNext = (e) => {
+    const errors = []
+
+    switch (currentPage) {
+      case "profile-info":
+        const { fullName, designation, summary } = resumeData.profileInfo
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)